tests/FakeItEasy.Specs/WrappingFakeSpecs.cs

```diff
@@ -272,6 +274,56 @@ public static void NotAWrappingFakeCallsWrappedMethod(IFoo fake, Exception excep
                     .WithMessage("The configured fake is not a wrapping fake."));
         }
 
+        [Scenario]
+        public static void EventSubscriptionSubscribesToWrappedEvent(Foo realObject, IFoo wrapper, EventHandler handler)
+        {
+            "Given a real object"
+                .x(() => realObject = new Foo());
+
+            "And a fake wrapping this object"
+                .x(() => wrapper = A.Fake<IFoo>(o => o.Wrapping(realObject)));
+
+            "And an event handler"
+                .x(() => handler = A.Fake<EventHandler>());
+
+            "When the handler is subscribed to the fake's event"
+                .x(() => wrapper.SomeEvent += handler);
+
+            "And the event is raised using a method on the wrapped object"
+                .x(() => realObject.RaiseSomeEvent());
+
+            "Then the handler is called"
+                .x(() => A.CallTo(handler).MustHaveHappened());
+        }
+
+        [Scenario]
+        public static void RaisingEventOnWrappingFakeUsingRaiseSyntax(Foo realObject, IFoo wrapper, EventHandler handler, Exception exception)
+        {
+            "Given a real object"
+                .x(() => realObject = new Foo());
+
+            "And a fake wrapping this object"
+                .x(() => wrapper = A.Fake<IFoo>(o => o.Wrapping(realObject)));
+
+            "And an event handler"
+                .x(() => handler = A.Fake<EventHandler>());
+
+            "When the handler is subscribed to the fake's event"
+                .x(() => wrapper.SomeEvent += handler);
+
+            "And the event is raised using the '+= Raise' syntax"
+                .x(() => realObject.SomeEvent += Raise.WithEmpty());

```

Comment:
Do we intend to try to raise the event from the `realObject`? This is an error, no? I think the more interesting behaviour is if we try to `Raise` from the Fake.

Reply:
Yes. In fact, I changed that spec, but didn't yet commit the changes. The new spec ensures that using `+= Raise` on the fake throws a NotSupportedException (as opposed to throwing when the dummy handler is called)